src/app/views/example/overview/table/table.component.html

```diff
@@ -57,8 +57,8 @@
     <terra-no-result-notice
         *ngIf="_dataSource.data.length === 0"
         [inputButtons]="_noResultButtons"
-        [inputTextPrimary]="'noResult' | translate: lang"
-        [inputTextSecondary]="'test' | translate: lang"
+        [inputTextPrimary]="(inputTextPrimary | translate: lang) ?? ''"
```

Comment:
Das ergibt irgendwie keinen Sinn. Was genau wurde hier bemängelt?

Reply:
Folgendes: ![Bildschirmfoto 2021-07-23 um 08 08 33](https://user-images.githubusercontent.com/47743312/126743412-62c21d32-c899-47f7-a347-0dbec52ea9f9.png)

Vermutlich gleiches Problem, dass sie `null` zurück geben kann? 😅

Reply:
Das auslagern von dem Key in eine Variable bringt auf jeden fall nichts. Das kannste wieder rückgängig machen^^

Reply:
done